website/pages/docs/integrate/nia-integration.mdx

```diff
@@ -44,14 +44,14 @@ Consul-Terraform-Sync compatible Terraform module development process is fairly
 - Consul-Terraform-Sync use case [documentation](https://www.consul.io/use-cases/network-infrastructure-automation)
 - Writing Consul-Terraform-Sync compatible Terraform modules [guide](Will link to NIA Tech Preview section - Requirements - Network Integration - How to Create a Module)
 - Example [module](https://registry.terraform.io/modules/findkim/ngfw/panos/0.0.1-beta3) for reference
-- Contributing to Terraform module registry [guidelines](https://www.terraform.io/docs/registry/modules/publish.html)
+- Publishing to Terraform Registry [guidelines](https://www.terraform.io/docs/registry/modules/publish.html)
 
 ### 3. Develop & Test
 
-Terraform modules are written in HashiCorp Configuration Language (HCL). Writing [Terraform modules](https://www.terraform.io/docs/modules/index.html) or a [tutorial to build a module](https://learn.hashicorp.com/tutorials/terraform/module-create) are a good resource to begin writing a new module.
-Consul-Terraform-Sync compatible modules follow the [standard module structure](https://www.terraform.io/docs/modules/index.html#standard-module-structure). Modules can use syntax supported by Terraform version 0.13, or higher. Consul-Terraform-Sync agent is designed to integrate with any module that satisfies these [specifications](https://docs.google.com/document/d/1eP918lf3lu-VVk4pBt4zVJ-eb4gNI6XBhpvZ3UnKNLw/edit#bookmark=id.f7kafih7a9qo). The guide will give you an introduction of the code structure and the basics of authoring a plugin that Terraform can interact with. Additionally developers are encouraged to use the [PAN-OS NGFW](https://registry.terraform.io/modules/findkim/ngfw/panos/0.0.1-beta3) module as an implementation reference.
+Terraform modules are written in HashiCorp Configuration Language (HCL). Writing [Terraform modules](https://www.terraform.io/docs/modules/index.html) or a [tutorial to build a module](https://learn.hashicorp.com/tutorials/terraform/module-create) are good resources to begin writing a new module.
+Consul-Terraform-Sync compatible modules follow the [standard module structure](https://www.terraform.io/docs/modules/index.html#standard-module-structure). Modules can use syntax supported by Terraform version 0.13, or higher. Consul-Terraform-Sync is designed to integrate with any module that satisfies these [specifications](https://docs.google.com/document/d/1eP918lf3lu-VVk4pBt4zVJ-eb4gNI6XBhpvZ3UnKNLw/edit#bookmark=id.f7kafih7a9qo). The guide will give you an introduction of the code structure and the basics of authoring a module for Terraform. Additionally developers are encouraged to use the [PAN-OS NGFW](https://registry.terraform.io/modules/findkim/ngfw/panos/0.0.1-beta3) module as an implementation reference.
```

Comment:
The term plugin in the Terraform space is strongly associated with providers and provisioners, so was looking to rephrase it.

Reply:
👍 Thank you Kim!